docs/assets/head-content.html

```diff
@@ -0,0 +1,30 @@
+<style>
+  .link-shinylive {
+    position: absolute;
+    bottom: 0;
+    right: 0.5em;
+    background-color: unset;
+    font-family: var(--bs-font-sans-serif);
+  }
+</style>
+<script>
+  document.addEventListener("DOMContentLoaded", function () {
+    // Add shinylive links to code blocks
+    document.querySelectorAll("[data-shinylive]").forEach(function (el) {
+      let pre = el.querySelector("pre");
+      const link = document.createElement("a");
+      link.classList.add("link-shinylive");
+      link.target = "_blank";
+      link.rel = "noopener noreferrer";
+      link.href = el.dataset.shinylive;
+      link.innerHTML = `<i class="bi bi-lightning-fill"></i> Run on shinylive`;
+      pre.appendChild(link);
+    });
+    // Add nav-underline to panel-underline where appropriate
+    document.querySelectorAll(".panel-underline .nav").forEach((x) => {
+      x.classList.remove("nav-tabs");
+      x.classList.add("nav-underline");
+      x.classList.add("justify-content-center");
+    });
+  });
```

Comment:
TODO: this should probably get updated/removed in favor of infrastructure in #66